test/unit/metrics/test_metrics_engine_dicts.py

```diff
@@ -88,6 +88,27 @@
     metrics.log_loss_group_min: 7.67546133,
 }
 
+classification_keys = [
```

Comment:
is this used anywhere?

Reply:
In a previous attempt at writing the tests :-)